mono_repo/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.2.0
```

Comment:
cool to release right away – just wait for my yaml tweak!

Reply:
I beat you – just make sure you rebase and rerun – and see the better pretty output!

Reply:
cheater :p

Reply:
Indeed!